### PR TITLE
Regions

### PR DESCRIPTION
**Pull request by alaahd**
**Description**

_Write a short description of the changes included in this pull request._

**Checklist**

* [ ] There is a related GitHub issue
* [ ] Tests are included if applicable
* [ ] Any added code is fully documented

**Issue**

Resolves #???

### CODE REVIEW

**Comment by alaahd** [comment]
I have read the CLA Document and I hereby sign the CLA

File: app/components/UI/FiatOnRampAggregator/Views/Region.js [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+		marginLeft: 7,
+	},
+});
+
+const Region = () => {
+	const [rememberRegion, setRememberRegion] = useState(false);
+	const [isTokenSelectorModalVisible, toggleTokenSelectorModal, , hideTokenSelectorModal] = useModalHandler(false);
+	const { setSelectedCountry, setSelectedRegion } = useFiatOnRampSDK();
+	// eslint-disable-next-line no-unused-vars
+	const [showAlert, setShowAlert] = useState(false);
```
Comment (wachunei) [inline app/components/UI/FiatOnRampAggregator/Views/Region.js:39]: Are we going to use this alert in the future?

Comment (alaahd) [reply app/components/UI/FiatOnRampAggregator/Views/Region.js:39]: Yes we will!

Comment (wachunei) [reply app/components/UI/FiatOnRampAggregator/Views/Region.js:39]: Ok 👌

File: app/components/UI/FiatOnRampAggregator/components/RegionModal.tsx [hunk 2]
```diff
@@ -0,0 +1,351 @@
...
+		flexDirection: 'row',
+		alignItems: 'center',
+	},
+	emoji: {
+		paddingRight: 16,
+	},
+});
+
+const Separator = () => <View style={styles.separator} />;
+const MAX_REGION_RESULTS = 20;
```
[the remark this answers is not in the document]
Comment (VikChawla) [reply app/components/UI/FiatOnRampAggregator/components/RegionModal.tsx:92]: Should I move both constants? or just MAX_REGION_RESULTS

[the remark this answers is not in the document]
Comment (wachunei) [reply app/components/UI/FiatOnRampAggregator/components/RegionModal.tsx:92]: just `MAX_REGION_RESULTS`

[the remark this answers is not in the document]
Comment (alaahd) [reply app/components/UI/FiatOnRampAggregator/components/RegionModal.tsx]: fixed

File: app/components/UI/FiatOnRampAggregator/Views/Region.js [hunk 3]
```diff
@@ -40,7 +40,8 @@ const Region = () => {
 	const [{ data, isFetching, error }] = useSDKMethod('getCountries');
 	const [selectedRegionObject, setSelectedRegionObject] = useState({});
 	const navigation = useNavigation();
-	const handleCheckBox = () => {
+
+	const handleChangeRememberRegion = () => {
 		setRememberRegion(!rememberRegion);
```
Comment (wachunei) [inline app/components/UI/FiatOnRampAggregator/Views/Region.js:45]: Use arrow function setState to flip booleans.
```suggestion
		setRememberRegion(currentRememberRegion => !currentRememberRegion);
```

Comment (alaahd) [reply app/components/UI/FiatOnRampAggregator/Views/Region.js]: That was an eagle-eye comment :) love it. changed

**Pull request merged**